packages/fluentui/react-northstar/src/components/SplitButton/SplitButton.tsx

```diff
@@ -112,16 +118,32 @@ class SplitButton extends AutoControlledComponent<WithAsProp<SplitButtonProps>,
     primary: customPropTypes.every([customPropTypes.disallow(['secondary']), PropTypes.bool]),
     secondary: customPropTypes.every([customPropTypes.disallow(['primary']), PropTypes.bool]),
     toggleButton: customPropTypes.itemShorthand,
+    position: PropTypes.oneOf(POSITIONS),
+    align: PropTypes.oneOf(ALIGNMENTS),
+    flipBoundary: PropTypes.oneOfType([
+      PropTypes.object as PropTypes.Requireable<Element>,
+      PropTypes.oneOf<'scrollParent' | 'window' | 'viewport'>(['scrollParent', 'window', 'viewport']),
+    ]),
+    overflowBoundary: PropTypes.oneOfType([
+      PropTypes.object as PropTypes.Requireable<Element>,
+      PropTypes.oneOf<'scrollParent' | 'window' | 'viewport'>(['scrollParent', 'window', 'viewport']),
+    ]),
+    positionFixed: PropTypes.bool,
+    offset: PropTypes.string,
+    unstable_pinned: PropTypes.bool,
   };
 
   static defaultProps = {
     accessibility: splitButtonBehavior,
-    as: 'div',
     toggleButton: {},
+    position: 'below',
+    align: 'start',
   };
 
   static autoControlledProps = ['open'];
 
+  targetRef = React.createRef<HTMLButtonElement>();
```

Comment:
```suggestion
  targetRef = React.createRef<HTMLDivElement>();
```

I think that container is a `div`...